README.md

```diff
@@ -54,6 +54,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |21| Personal Tab with Nav-Bar menu | Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app. ||[View][tab-navbar-menu#ts]|
 |22| App Caching in Meetings | This sample shows how to improve the subsequent loading time of an App. |||[View][app-cache-meetings#nodejs]|
 |23| Personal Tab with Nav-Bar menu | Add multiple actions to the upper right in Nav-Bar and build an overflow menu for extra actions in an app. ||[View][tab-navbar-menu#ts]|
+|24| Tab User Profile In Outlook | Opens a profile card at a specified position to show profile information about a persona. ||[View][tab-user-profile#ts]|
```

Comment:
Opens a profile card at a specified position to show profile information about a persona. =>

This sample app shows how to open a user profile information card at a specified position in Outlook using personal tab app.

Reply:
Fixed